tests/model_selection/dask_searchcv/test_model_selection.py

```diff
@@ -975,21 +976,23 @@ def test_mock_with_fit_param_raises():
         clf.fit(X, y)
 
 
-@pytest.mark.parametrize("metric,greater_is_better,needs_proba,sample_wt", [
-    (accuracy_score, True, False, [1, 999999, 1, 999999]),
-    (accuracy_score, True, False, [100000, 200000, 100000, 200000]),
-    (accuracy_score, True, False, [100000, 100000, 100000, 100000]),
-    (accuracy_score, True, False, [200000, 100000, 200000, 100000]),
-    (accuracy_score, True, False, [999999, 1, 999999, 1]),
-    (accuracy_score, True, False, [2000000, 1000000, 1, 999999]),
-    (log_loss, False, True, [2500000, 500000, 200000, 100000]),
-    (brier_score_loss, False, True, [2500000, 500000, 200000, 100000]),
+@pytest.mark.parametrize("metric,greater_is_better,needs_proba,estimator,estimator_params,sample_wt", [
+    (accuracy_score, True, False, LogisticRegression(), {"random_state": [15432], "solver": ["lbfgs"]}, [1, 999999, 1, 999999]),
```

Comment:
I would create a `IMP_WT_LOG_REG_PARAMS` constant just above the test like the following:

```python
IMP_WT_LOG_REG_PARAMS = {
    "solver":         "lbfgs",
    "penalty":        "l2",
    "tol":            1e-6,
    "max_iter":       10000,
    "fit_intercept":  False,
    "random_state":   42
}
```

Then create the logistic regression models in the test cases as:

```python
LogisticRegression(**IMP_WT_LOG_REG_PARAMS)
```

This will provide many benefits like locking the random seed if any randomization is used, guarantee a fitting algorithm and attempt to make the model overfit as much as possible.  This will help ensure that the tests can increase the numeric precision in the calls to `np.testing.assert_almost_equal` in the test.

Reply:
added